sphinx/getting_started.md

```diff
@@ -18,67 +18,90 @@ pip install guppylang
 
 Guppy can be used with Python versions 3.10, 3.11, 3.12 and 3.13. The MacOS, Linux and Windows operating systems are supported.
 
-The source code for Guppy can be found in a public repository on [github](https://github.com/quantinuum/guppylang). If you have a feature request or think you have found a bug, feel free to raise a [github issue](https://github.com/quantinuum/guppylang/issues).
+The source code for Guppy can be found in a public repository on [GitHub](https://github.com/quantinuum/guppylang). If you have a feature request or think you have found a bug, feel free to raise a [GitHub issue](https://github.com/quantinuum/guppylang/issues).
 
 Guppy programs can be executed on the [Selene](https://github.com/quantinuum/selene) emulator. As of the v0.21 release, Selene is now included with `guppylang` and powers the [guppylang.emulator](../sphinx/api/emulator.md) module under the hood.
 
-## Example: Quantum Teleportation
+## Example: A simple circuit
 
-As a first example, let's write a Guppy program to implement the [quantum teleportation](https://en.wikipedia.org/wiki/Quantum_teleportation) scheme.
+As a first example, let's write a Guppy program to implement a simple quantum circuit.
+This circuit is deliberately designed to involve some classical control based on measurement outcomes:
+![circuit](./_static/getting_started_circuit.png)
```

Comment:
Its great that you have annotated the states at the different timesteps of the circuit. 

Maybe for total clarity it would be good to explicitly state what the probabilities of measuring $|00\rangle$ and $|10\rangle$ are. Perhaps this would make it clearer for beginners why the plot below is correct.

Reply:
<img width="1268" height="490" alt="image" src="https://github.com/user-attachments/assets/f20776d8-79de-48b7-9c3d-63292081919d" />